addons/adyen_platforms/controllers/adyen_platforms.py

```diff
@@ -1,13 +1,46 @@
-# -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
+import json
+import logging
+from pprint import pformat
+
 from odoo import http
 from odoo.http import request
 
+from odoo.addons.adyen_platforms.util import odoo_payments_proxy_control
+
+_logger = logging.getLogger(__name__)
+
 
 class AdyenPlatformsController(http.Controller):
 
     @http.route('/adyen_platforms/create_account', type='http', auth='user', website=True)
     def adyen_platforms_create_account(self, creation_token):
         request.session['adyen_creation_token'] = creation_token
         return request.redirect('/web?#action=adyen_platforms.adyen_account_action_create')
+
+    @odoo_payments_proxy_control()
+    @http.route('/adyen_platforms/account_notification', type='json', auth='public', csrf=False)
+    def adyen_platforms_notification(self):
+        data = json.loads(request.httprequest.data)
```

Comment:
`Request.data` is soft-deprecated by werkzeug as it always attempts to parse the form data.

Also the request data is already fetched, parsed, and cached on `request.jsonrequest`. That avoids risks of misdecoding, as `json.loads` will accept bytes (from 3.6) but only if it's one of the UTFs.